CONTRIBUTING.md

```diff
@@ -0,0 +1,176 @@
+How to contribute
+-----------------
+
+The preferred workflow for contributing to the OpenML python connector is to
+fork the [main repository](https://github.com/openml/openml-python) on
+GitHub, clone, check out the branch `develop`, and develop on a new branch
+branch. Steps:
+
+1. Fork the [project repository](https://github.com/openml/openml-python)
+   by clicking on the 'Fork' button near the top right of the page. This creates
+   a copy of the code under your GitHub user account. For more details on
+   how to fork a repository see [this guide](https://help.github.com/articles/fork-a-repo/).
+
+2. Clone your fork of the openml-python repo from your GitHub account to your
+local disk:
+
+   ```bash
+   $ git clone git@github.com:YourLogin/openml-python.git
+   $ cd openml-python
+   ```
+
+3. Swith to the ``develop`` branch:
+
+   ```bash
+   $ git checkout develop
+   ```
+
+3. Create a ``feature`` branch to hold your development changes:
+
+   ```bash
+   $ git checkout -b my-feature
```

Comment:
I prefer using the branch type as suffix to keep the overview, i.e.: `feature/idea` or `feature-idea`.